src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -196,13 +207,29 @@ def _calculate_percentiles(self, data: List[Union[int, float]], attr: str) -> No
 
     def _calculate_minmax(self, data: List[Union[int, float]], attr: str) -> None:
         min, max = np.min(data), np.max(data)
+        self._stats_dict[attr]["max"] = str(max)
+        self._stats_dict[attr]["min"] = str(min)
         setattr(self, "min_" + attr, min)
         setattr(self, "max_" + attr, max)
 
     def _calculate_std(self, data: List[Union[int, float]], attr: str) -> None:
         std = np.std(data)
+        self._stats_dict[attr]["std"] = str(std)
         setattr(self, "std_" + attr, std)
 
+    def _add_units(self, key) -> None:
+        if self._is_time_field(key):
+            self._stats_dict[key]["unit"] = "ns"
+        if key == "request_throughput":
+            self._stats_dict[key]["unit"] = "request per sec"
```

Comment:
Would `request/s` be more consistent with `ns`? 
- `request/sec` <--> `nsec`
- `request/s` <--> `ns`

(same as token/s units below).

Reply:
Updated